Sources/SwiftDocCUtilities/CatalogTemplates/Article.swift

```diff
@@ -0,0 +1,34 @@
+/*
+ This source file is part of the Swift.org open source project
+
+ Copyright (c) 2021-2023 Apple Inc. and the Swift project authors
+ Licensed under Apache License v2.0 with Runtime Library Exception
+
+ See https://swift.org/LICENSE.txt for license information
+ See https://swift.org/CONTRIBUTORS.txt for Swift project authors
+*/
+
+import Foundation
+import SwiftDocC
+
+struct Article {
```

Comment:
I think we want a different name that is more specific to what this is, maybe ```ArticleTemplate```

Reply:
Nice suggestion, introduced here https://github.com/apple/swift-docc/pull/719/commits/56077e17bc599347e188d7802186ff56c7d5bb64